content/agent/_index.md

```diff
@@ -27,18 +33,114 @@ The Datadog Agent is software that runs on your hosts. It collects events and me
 
 {{< partial name="platforms/platforms.html" >}}
 
-The Agent has three main parts: the **collector**, **DogStatsD**, and the **forwarder**:
+## Agent Architecture
+
+{{< tabs >}}
+{{% tab "Agent v6" %}}
+
+Agent v6 is a complete rewrite in Go of the Agent v5. V6 offers better performances, smaller footprint, and more features. It is the default Datadog Agent (v5 is no longer in active development).
+
+Agent v6 is a composed of a main process responsible for collecting infrastructure metrics, logs, and receiving DogStatsD metrics. The main components to this process are:
+
+* The Collector is in charge of running checks and collecting metrics.
+* The Forwarder sends payloads to Datadog.
+
+Two optional processes are spawned by the Agent if enabled in the `datadog.yaml` configuration file:
+
+* The APM Agent is a process to collect traces (enabled by default).
+* The Process Agent is a process to collect live process information. By default, it only collects available containers, otherwise it is disabled.
+
+On Windows the services are listed as:
+
+| Service               | Description             |
+|-----------------------|-------------------------|
+| DatadogAgent          | “Datadog Agent”         |
+| datadog-trace-agent   | “Datadog Trace Agent”   |
+| datadog-process-agent | "Datadog Process Agent” |
+
+By default the Agent binds 3 ports on Linux and 4 on Windows and OSX:
+
+| Port | Description                                                                                 |
+|------|---------------------------------------------------------------------------------------------|
+| 5000 | Exposes runtime metrics about the Agent.                                                    |
+| 5001 | Used by the Agent CLI and GUI to send commands and pull information from the running Agent. |
+| 5002 | Used by the GUI server on Windows and OSX.                                                  |
```

Comment:
This port actually is the GUI server. So I would say: `use to serve the GUI on Windows and OSX.`